mixins/localize/test/localize-mixin.test.js

```diff
@@ -246,12 +247,12 @@ const staticTag = defineCE(StaticEl);
 
 const asyncTag = defineCE(
 	class extends StaticEl {
-		static getLocalizeResources(langs) {
+		static async getLocalizeResources(langs) {
 			for (let i = 0; i < langs.length; i++) {
 				if (this.langResources[langs[i]]) {
 					const resources = {
 						language: langs[i],
-						resources: this.langResources[langs[i]]
+						resources: { getsConcatenated: 'I‘m here too!' }
```

Comment:
This was casusing confusion because it was already extending a class with the same resources fetched synchronously.